pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1426,8 +1426,11 @@ func buildHTTPConnectionManager(node *model.Proxy, env *model.Environment, httpO
 			Name: xdsutil.FileAccessLog,
 		}
 
-		if util.IsXDSMarshalingToAnyEnabled(node) {
+		if util.IsProxyVersionGE11(node) {
 			buildAccessLog(fl, env)
+		}
```

Comment:
this is wrong.. please follow logic here: https://github.com/istio/istio/blob/master/pilot/pkg/networking/core/v1alpha3/listener.go#L1421

Reply:
Sorry I'm not sure what you're saying is wrong here...